### PR TITLE
pkg: minor code style improvements

### PR DESCRIPTION
**Pull request by jiangliu**
Minor code style improvements, should be no functional changes.

### CODE REVIEW

File: pkg/daemon/daemon.go [hunk 1]
```diff
@@ -60,16 +60,17 @@ type Daemon struct {
 	States States
 
 	mu sync.Mutex
-	// FsInstances map[int]*Rafs
-	// should be persisted to DB
-	// maps to at least one rafs instance.
-	// It is possible to be empty after the daemon object is created.
+	// Host all RAFS filesysstems managed by this daemon:
```
Comment (changweige) [inline pkg/daemon/daemon.go:63]: Typo filesystems ?

File: pkg/system/system.go [hunk 2]
```diff
@@ -148,6 +148,21 @@ func NewSystemController(fs *filesystem.Filesystem, manager *manager.Manager, so
 	return &sc, nil
 }
 
+func (sc *Controller) Run() error {
```
Comment (changweige) [inline pkg/system/system.go:151]: Um, why several functions like this `Run` are moved within the same source file, which makes the review a little harder?

File: pkg/daemon/rafs.go [hunk 3]
```diff
@@ -95,18 +94,18 @@ func (rs *rafsSet) List() map[string]*Rafs {
 	return instances
 }
 
-func (rs *rafsSet) ListUnlocked() map[string]*Rafs {
+func (rs *rafsSet) ListLocked() map[string]*Rafs {
```
Comment (changweige) [inline pkg/daemon/rafs.go:97]: `Locked` suffix indicates this function has lock inside its body?

Comment (jiangliu) [reply pkg/daemon/rafs.go:97]: The `Locked` suffix means this function should be called with related lock held. It's a code style from Linux kernel:)

File: pkg/system/system.go (hunk 2 above)
Comment (jiangliu) [reply pkg/system/system.go:151]: I'm trying to group logically related functions near to each other.

File: pkg/daemon/daemon.go (hunk 1 above)
Comment (jiangliu) [reply pkg/daemon/daemon.go:63]: Fixed

**Review by imeoer — APPROVED** [review]
LGTM.

**Pull request merged**